### PR TITLE
feat(editor-document): add .properties getter, add .updateProperty method

### PR DESCRIPTION
**Pull request by ilyamore88**

### CODE REVIEW

File: src/entities/EditorDocument/index.ts [hunk 1]
```diff
@@ -80,6 +80,23 @@ export class EditorDocument {
...
+    return this.#properties;
+  }
+
+  /**
+   * Updates a property of the EditorDocument.
+   *
+   * @param name - The name of the property to update
+   * @param value - The value to update the property with
+   */
+  public updateProperty(name: PropName, value: unknown): void {
```
Comment (neSpecc) [inline src/entities/EditorDocument/index.ts:96]: maybe setProperty?

File: src/entities/EditorDocument/index.ts [hunk 2]
```diff
@@ -80,6 +80,23 @@ export class EditorDocument {
...
+    this.#properties[name] = value;
```
Comment (neSpecc) [inline src/entities/EditorDocument/index.ts:97]: could you write a test case of passing unexisting property name?

Comment (ilyamore88) [reply src/entities/EditorDocument/index.ts:108]: Done

**Review by gohabereg — APPROVED** [review]

File: src/entities/EditorDocument/EditorDocument.spec.ts [hunk 3]
```diff
@@ -246,4 +246,80 @@ describe('EditorDocument', () => {
...
+
+      expect(document.properties[propertyName]).toBe(expectedValue);
+    });
+
+    it('should add the property if it does not exist', () => {
+      const propertyName = 'readOnly';
+      const expectedValue = true;
+      const document = new EditorDocument({
+        children: [],
+        properties: {},
```
Comment (neSpecc) [inline src/entities/EditorDocument/EditorDocument.spec.ts:317]: I think, `properties` should be optional in construction params. And set as an empty object in a constructor.

Comment (ilyamore88) [reply src/entities/EditorDocument/EditorDocument.spec.ts:317]: Made it optional. But for these test cases, I want to have an empty object in the constructor for more readability.

**Review by neSpecc — APPROVED** [review]

**Pull request merged**